api/tests/functional-tests/crud/test_create_delete.py

```diff
@@ -1316,13 +1316,8 @@ def test_create_clf_metrics(
     )
     assert len(resp) == 1
     resp = resp[0]
-    missing_pred_keys = resp.missing_pred_keys
```

Comment:
these values are always empty lists for classification tasks now, so these tests aren't necessary